src/libpmem2/aarch64/init.c

```diff
@@ -38,10 +71,15 @@ pmem2_arch_init(struct pmem2_arch_info *info)
 	LOG(3, NULL);
 
 	info->fence = memory_barrier;
-	info->flush = flush_dcache;
+	if (is_dc_pop_available())
```

Comment:
In the future, this should also include a check for ARM's eADR equivalent. But for now, it's fine.